shared/js/background/chrome-events.es6.js

```diff
@@ -47,6 +48,14 @@ chrome.webRequest.onHeadersReceived.addListener(
     }
 )
 
+chrome.webRequest.onHeadersReceived.addListener(
+    https.setUpgradeInsecureRequestHeader,
```

Comment:
Just posting a comment so this doesn't get lost - as we discussed over chat, this needs to not happen if the site is whitelisted.